### PR TITLE
Allow priority to be set as a config option for delayed job backend.

### PR DESCRIPTION
**Pull request by gspiers**
I was looking for a way to pass down the priority that the processing job should be scheduled with when using the delayed job backend. I added it as one of the arguments that can be passed for configuration of the backend.

If priority is not set it will, as it did before, schedule the job with the default delayed job priority.

### CODE REVIEW

File: lib/backgrounder/support/backends.rb [hunk 1]
```diff
@@ -23,10 +23,16 @@ def enqueue_for_backend(worker, class_name, subject_id, mounted_as)
           private
 
           def enqueue_delayed_job(worker, *args)
+            worker_args = {}
             if ::Delayed::Job.new.respond_to?(:queue)
-              ::Delayed::Job.enqueue worker.new(*args), :queue => queue_options[:queue]
+              worker_args[:queue] = queue_options[:queue] if queue_options[:queue]
+              # Only include priority if set
```
Comment (lardawge) [inline lib/backgrounder/support/backends.rb]: Can we remove this comment.

**Comment by gspiers** [comment]
Removed the comments as requested, let me know if there's anything else. Thanks :)

**Comment by axlekb** [comment]
I'd love to have this!

**Comment by lardawge** [comment]
Done. Please test against master and I will release a gem.

**Comment by gspiers** [comment]
@lardawge Sorry for the delay, I've only just got it into one of our releases for testing. Works perfectly.

Thanks again :+1:

**Pull request merged**